addons/mail/static/tests/discuss/call/call.test.js

```diff
@@ -320,3 +320,24 @@ test("'Start a meeting' in mobile", async () => {
     await click("[title='Show Member List']");
     await contains(".o-discuss-ChannelMember", { text: "Partner 2" });
 });
+
+test("mobile: displaying push to talk button", async () => {
```

Comment:
`show Push-to-Talk button on mobile`

In general a complete sentence is best. Avoid `:` if you can (makes naming of test more complex), or `ing`-ing verbs which also makes it harder to grasps the idea (`-ing` verbs usually refers to an ongoing action, so it feels like we miss something by reading a sentence with `-ing` in it).